client/gutenberg/editor/utils.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Internal dependencies
+ */
+import { requestHttpData } from 'state/data-layer/http-data';
+import { WPCOM_HTTP_REQUEST } from 'state/action-types';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+/**
+ * Request a site's frame nonce from the v1.3 endpoint.
+ *
+ * @param {number} siteId  Site Id.
+ * @return {*} Stored data container for request.
+ */
+export const getFrameNonce = siteId =>
+	requestHttpData(
+		`frame-nonce-${ siteId }`,
+		http( {
+			type: WPCOM_HTTP_REQUEST,
+			method: 'GET',
+			path: `/sites/${ siteId }`,
+			apiVersion: '1.3',
+		} ),
+		{
+			fromApi: () => {},
+			freshness: 0,
```

Comment:
If you never want to cache this, you'll need special handling to clear the cache:

For example:
```js
import { empty } from 'state/data-layer/http-data';

httpData.set( `frame-nonce-${ siteId }`, empty );

//then make the request
//requestHttpData( ...
```

Reply:
We can set freshness to the freshness of the nonce. Better yet, if the API can return the nonce expiration then we can automatically expire our cache based on that

Reply:
I'm pretty sure that's not returned in the API response, but I think it'd be okay to hard code that and folks can leave a comment of where to find the related expiry logic on the API side.

Reply:
A smaller interval than the total expiry time would be okay too.

Reply:
Tested this locally, and freshness of 0 is not advised unless we want to move the call to componentDidMount or similar. mapStateToProps may be called many times during a component render.